test-suite/src/test/scala/org/scalajs/testsuite/javalib/FormatterTest.scala

```diff
@@ -189,6 +189,18 @@ object FormatterTest extends JasmineTest {
       expectF("%f",      new JDouble(3e10)).toEqual("30000000000.000000")
       expectF("%04f",    new JDouble(Double.NaN)).toEqual(" NaN")
     }
+    
+    it("should support ',' flag") {
+      expectF("%,d",   new Integer(0)).toEqual("0")
+      expectF("%,d",   new Integer(5)).toEqual("5")
+      expectF("%,d",   new Integer(50)).toEqual("50")
+      expectF("%,d",   new Integer(500)).toEqual("500")
+      expectF("%,.2f",   new JFloat(500)).toEqual("500.00")
+      
+      expectF("%,d",   new Integer(5000)).toEqual("5,000")
+      expectF("%,d",   new Integer(500000)).toEqual("500,000")
+      expectF("%,.2f",   new JFloat(5000)).toEqual("5,000.00")
```

Comment:
Also, the indentation is strange here. Either, align all `new` calls, or use a single space. (Convention in this file seems to be to align though).